TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/transform/IAuthTransformerFactory.java

```diff
@@ -2,6 +2,8 @@
 
 import java.io.IOException;
 
+import org.opensearch.migrations.IHttpMessage;
+
 public interface IAuthTransformerFactory extends AutoCloseable {
     IAuthTransformer getAuthTransformer(IHttpMessage httpMessage);
```

Comment:
I wouldn't use IHttpMessage here. This package only needs to deal with HttpJsonMessageWithFaultingPayload objects (which is also what you've spec'ed the function out for).